intl.emu

```diff
@@ -23,6 +54,8 @@
           ): either a normal completion containing an Intl mathematical value or a throw completion
         </h1>
         <dl class="header">
+          <dt>skip global checks</dt>
+          <dd>true</dd>
```

Comment:
what's this for?

Reply:
This is needed because ecmarkup complains about the `ToIntlMathematicalValue` AO never being used in a way that causes it to throw an exception. It complains that its signature should be changed. Here's the warning it emits if I drop this change:

> warning: every call site of ToIntlMathematicalValue asserts the return value is a normal completion; it should be refactored to not return a completion record at all. if this AO is called in ways ecmarkup cannot analyze, add the "skip global checks" attribute to the header (always-asserted-normal) at intl.emu:46:7:

Reply:
The change there is intended to be temporary. I indicated as much in the commit. I added that inline now, in the text, as an HTML comment.